### DISCUSSION TITLE
Invertierter Text

### DISCUSSION THREAD

**Discussion by ChrisFTB** [olikraus/u8g2#1660] [Q&A] [answered]
Nabend zusammen!
Ich knobel jetzt schon drei Tage rum, aber mein Display treibt mich noch in den Wahnsinn!

Ich habe ein 256x64 OLED mit 1322 Controller.

Mein Ziel ist eigentlich nichts wildes: ich möchte ein Wort mittig auf dem Display. Allerdings invertiert- sprich: Alle Pixel leuchten außer die Textpixel.

Die "normale" Version habe ich schon hinbekommen Dank Olis Library.... aber der letzte Meter will einfach nicht.....

Hätte jemand einen Tipp?

Vielen Dank,
Chris

**Comment by olikraus** [comment, marked as answer]
Mit der drawButton Funktion müsste es gehen.
Andere Möglichkeit: Box zeichnen und dann mit Draw color 0 den Text darüber schreiben.
